server/odc-service/src/main/java/com/oceanbase/odc/service/task/util/JobUtils.java

```diff
@@ -132,14 +133,15 @@ public static boolean isProcessRunModeOfEnv() {
     public static ConnectionConfig getMetaDBConnectionConfig() {
         ConnectionConfig config = new ConnectionConfig();
         config.setHost(System.getProperty(JobEnvKeyConstants.ODC_EXECUTOR_DATABASE_HOST));
```

Comment:
environment variable could expire while the main application is still running?

Reply:
may occur in process mode scenario, let's trace in separated issue